Python/Product/PythonTools/PythonTools/Extensions.cs

```diff
@@ -486,7 +493,14 @@ public static class Extensions {
                 IVsHierarchy hierarchy;
                 uint itemid;
                 docTable.GetDocumentHierarchyItem(cookie, out hierarchy, out itemid);
-                return hierarchy.GetProject()?.GetPythonProject();
+                var project = hierarchy.GetProject();
+                if (project != null) {
+                    return project.GetPythonProject();
+                }
+
+                object projectObj;
```

Comment:
GetProject is already getting ExtObject from the hierarchy...  So this code just seems to be completely redundant?